core/src/table/table.module.css

```diff
@@ -66,11 +66,11 @@
 .container th {
 	position: sticky;
 	top: 0;
-	z-index: 2;
+	z-index: var(--z-th);
 	border-bottom-width: 1px;
 	border-bottom-style: solid;
 }
 
 .container th:first-child {
-	z-index: 3;
+	z-index: calc(var(--z-th) + 1);
```

Comment:
This is quite dangerous for the future. It's better to define 2 z-index variables for th (e.g. `--z-th-1` and `--z-th-2`), instead of calculating here. The reason is because:
- Assume you want to have a component whose z-index is larger than table's header
- You go to the global css file and see that --z-th, then you use something that is larger than that (like +1)
- However, it wouldn't work, since we also have a "th:first-child" whose z-index is already +1 larger than --z-th

This means the better way to do this is to centralize *all* z-index, and only use the variables directly, so others can confidently look at the list and know which to use.

Reply:
yeah, I agree with you.